### ISSUE TITLE
Karl going towards obstacles

### ISSUE THREAD

**Issue by lucasb-eyer** [strands-project/strands_navigation#163]
Sometimes, Karl sees an obstacle, say front left of him, and instead of driving to the right (as according to the global plan), he decides to turn _left_, RIGHT INTO THE OBSTACLE. This is very strange behaviour I've been observing during the past previous weeks already.

Here is an example of it, the green arrows are the odometry, i.e. how he drove:

![goingtoobstacle](https://cloud.githubusercontent.com/assets/1476029/5164127/add00414-73d3-11e4-96cc-ba3a4ac9105e.png)

**Comment by lucasb-eyer** [comment]
By the way, backtracking got him out of the situation most of the time, but when the obstacle is a human, this is really awkward for that person...

**Comment by cdondrup** [comment]
We observe the same behaviour with Linda since we are in the museum. No idea what causes this, just saying that you're not alone ;)

**Comment by lucasb-eyer** [comment]
Good to know! The backtracking helps a lot, but it's more like curing the symptoms while we should fix the root cause.

**Comment by nilsbore** [comment]
Actually, I think this is related to the issue that we thought was caused by the updated drivers. This means that we can't use `move_base` to pass through doors, something that was quite reliable earlier. As it didn't get solved by a downgrade to the old drivers, we thought it had something to do with `dwa_local_planner` (since, as you say, the global plan looks good). We've tried to play around with a lot of the parameters there without any success. I also tried to downgrade to the earliest source compatible version of `dwa_local_planner` but that didn't solve anything. When the marathon is over I will try an even earlier version of `move_base` to see if there is any difference there.
